src/Materials/PBR/pbrSubSurfaceConfiguration.ts

```diff
@@ -353,6 +355,7 @@ export class PBRSubSurfaceConfiguration {
                     }
                 }
 
+                defines.SS_HAS_THICKNESS = this.minimumThickness !== 0.0 && this.maximumThickness !== 0.0;

```

Comment:
Why not doing `defines.SS_HAS_THICKNESS = (this.maximumThickness - this.minimumThickness) !== 0.0;` instead?

I'm not sure that `this.minimumThickness !== 0.0` should be a requirement(?)

Reply:
Quite right.